devTools/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "devtools",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "ts-node index.ts"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "sdk-sanity-generator": "git+ssh://git@github.com/CloudinaryLtd/sdk-sanity-generator.git",
```

Comment:
Keeping links to github.com/CloudinaryLtd in a public repo is kind of weird. nobody except us can run it... maybe we should move sdk-sanity-generator repo to cloudinary?

Reply:
I thought as well about making the library public.
I think there's no strong reason against it.

To be discussed.